libs/designsystem/checkbox/src/checkbox.component.ts

```diff
@@ -3,12 +3,14 @@ import {
   ChangeDetectionStrategy,
   Component,
   EventEmitter,
+  forwardRef,
   HostBinding,
   Input,
   Output,
 } from '@angular/core';
 import { IonicModule } from '@ionic/angular';
 import { UniqueIdGenerator } from '@kirbydesign/designsystem/helpers';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
```

Comment:
Please consider readability: order and grouping of imports: 
```suggestion
import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';

import { UniqueIdGenerator } from '@kirbydesign/designsystem/helpers';
```